### DISCUSSION TITLE
add config switch "add all available entities/sensors"

### DISCUSSION THREAD

**Discussion by bsafh** [kellerza/sunsynk#304] [Ideas]
I'd like to have a config switch "add all evailable entities" in the add-on config, insteak of having to go through the YAML definitions file for your inverter and copy-paste all sensors and switches which are not covered by the keywoeds "energy_management", "powerflow_card" and "settings" - or perhaps a keyword "all" ?
(see https://kellerza.github.io/sunsynk/reference/definitions )

**Comment by kellerza** [comment]
See - https://kellerza.github.io/sunsynk/reference/definitions#all-sensors
